polkadot/node/core/pvf/common/src/lib.rs

```diff
@@ -25,9 +25,18 @@ pub mod worker;
 
 pub use cpu_time::ProcessTime;
 
-// Used by `decl_worker_main!`.
-#[cfg(feature = "test-utils")]
-pub use sp_tracing;
+/// DO NOT USE - internal for macros only.
+#[doc(hidden)]
+pub mod __private {
+	pub mod sp_tracing {
+		pub fn try_init_simple() {
+			#[cfg(feature = "test-utils")]
+			{
+				let _ = ::sp_tracing::try_init_simple();
+			}
+		}
+	}
```

Comment:
When I see this. uhhh...
```suggestion
	#[cfg(feature = "test-utils")]
	pub use sp_tracing::try_init_simple;
```

Reply:
It is used in `decl_worker_main!` like this: `$crate::sp_tracing::try_init_simple();` which fails when the crate that uses the macro does not have the `test-utils` feature.  
I therefore added a stub in that case.

Reply:
You can see the compile error here @bkchr  https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3438147

Reply:
This stuff is even more broken than expected :roll_eyes:

Reply:
You think we can merge it like this?

Reply:
@ggwpez just remove the feature here entirely. And just expose it always. Is the macro feature gated at all?

Reply:
okay removed the feature gate.